Funcky.Async.Test/AsyncSequence/CycleRangeTest.cs

```diff
@@ -0,0 +1,68 @@
+using FsCheck;
+using FsCheck.Xunit;
+using Funcky.Async.Test.TestUtilities;
+using Funcky.Test.TestUtils;
+
+namespace Funcky.Async.Test;
+
+public sealed class CycleRangeTest
+{
+    [Fact]
+    public async Task CycleRangeIsEnumeratedLazilyAsync()
+    {
+        var doNotEnumerate = new FailOnEnumerateAsyncSequence<object>();
+
+        await using var cycleRange = AsyncSequence.CycleRange(doNotEnumerate);
+    }
+
+    [Fact]
+    public void CyclingAnEmptySetThrowsAnArgumentException()
+            => Assert.ThrowsAsync<InvalidOperationException>(CycleEmptySequenceAsync);
```

Comment:
This is indented too much

Reply:
Fixed in 0ffe8c5188b875c0c110579f527033f6657eebda